### ISSUE TITLE
Vite.js support

### ISSUE THREAD

**Issue by holtwick** [versatica/mediasoup-client#159]
## Bug Report

Running with [Vite](https://vitejs.dev/) causes an issue:

```
Uncaught TypeError: bowser.getParser is not a function
    detectDevice Device.js:35
    Device Device.js:135
    <anonymous> main.ts:5
```

### Your environment

- Operating system: macOS M1
- Browser version: Firefox 
- npm version:  7.5.3
- mediasoup-client version: 3.6.28

### Issue description

1. Set up a vite project (vue + ts): `npm init @vitejs/app`
2. Install mediasoup client: `npm i mediasoup-client`
3. To `main.ts` add these lines:

```ts
import * as mediasoupClient from 'mediasoup-client'
let mediasoupDevice = new mediasoupClient.Device()
```

**Expected:** Should not throw error

**Actual:** Throws error. Probably related to Vite's strict ESM module handling.

### Related
- #88 
- https://github.com/lancedikson/bowser/issues/339
- https://vitejs.dev/guide/dep-pre-bundling.html#dependency-pre-bundling

**Comment by ibc** [comment]
I don't know what Vite is and I don't have time to learn/try it. Can you add this into `mediasoup-client/src/Device.ts` in line 23?

```ts
(window as any).BOWSER = bowser;
```

then run `npm run typescript:build` in mediasoup-client folder, then include it into your Vite project, run it, open the browser and checkinspect/check what `window.BOWSER` looks like?

**Comment by ibc** [comment]
Even better, can you try the branch provided by this PR https://github.com/versatica/mediasoup-client/pull/160?

**Comment by holtwick** [comment]
Thanks a lot for looking into it! Very much appreciated. I described a fix in PR #160 and #161
